ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -15,10 +18,10 @@ if [[ "${CLUSTER_TYPE}" == "aws-arm64" ]]; then
   fi
   OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${ARM64_RELEASE_OVERRIDE}
   echo "Installing from initial release ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
-  openshift-install version
   # end of hack
 fi
 
+echo "Installer version: $(openshift-install version)"
```

Comment:
Can we drop this line and the earlier `openshift-install version` removal?  I've filed #20874 removing that whole arm64 block from this step, and explaining why I don't think it's useful here.

Reply:
removed